### DISCUSSION TITLE
Plausible Cloud and AGPL incompatibility

### DISCUSSION THREAD

**Discussion by mhitza** [plausible/analytics#992] [Anything Else]
> Notwithstanding any other provision of this License, **if you modify the Program, your modified version must prominently offer all users interacting with it remotely through a computer network (if your version supports such interaction) an opportunity to receive the Corresponding Source of your version by providing access to the Corresponding Source from a network server** at no charge, through some standard or customary means of facilitating copying of software. This Corresponding Source shall include the Corresponding Source for any work covered by version 3 of the GNU General Public License that is incorporated pursuant to the following paragraph.

Plausible Cloud is a modified version of the AGPL'ed Plausible analytics repository, thus the sources should be made available for that specific version of Plausible currently running on the official website.

**Comment by ukutaht** [comment]
It's not modified - we run exactly what's on the `stable` branch.

But I take your point that the code should be available from the network server itself. We should add a build step that zips up the source code and makes it available from an endpoint like `/code` or `/source` or something.

**Reply by mhitza** [reply]
I see. I've actually read the table in the readme and the blog post about open source development. For some reason they both lead me to believe source code was released in batches.

In terms of source code access the GitHub link that you have in the footer is enough.

**Reply by ukutaht** [reply]
Ah yeah I develop on this github repo on the `master` and `stable` branches. I just don't want to be cutting releases too often as it's an extra job that I have to do manually. That's why I batch them up into bigger releases every 6 months or so.
